### PR TITLE
update to 5.6.0-SNAPSHOT #795

### PR DESCRIPTION
**Pull request by btiwahorisu**
Please review #795

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>org.terasoluna.gfw.functionaltest</groupId>
   <artifactId>terasoluna-gfw-functionaltest</artifactId>
-  <version>5.5.1-SNAPSHOT</version>
+  <version>5.6.0-SNAPSHO</version>
```
Comment (hirakuriy) [inline pom.xml]: - [ ] fix `-SNAPSHO`

Comment (btiwahorisu) [reply pom.xml]: 記載の修正を行いました。
d2238f2276c07341148241ee721390bae5f4405b

Comment (hirakuriy) [reply pom.xml]: Please correct other pom.xml as well.

Comment (btiwahorisu) [reply pom.xml]: fix other pom.xml.
7bbb73dcd412fb26169f2b4a087d5cc88c674775

Comment (hirakuriy) [reply pom.xml]: Please fix here too.
https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/7bbb73dcd412fb26169f2b4a087d5cc88c674775/pom.xml#L6

**Pull request merged**